server/jetstream_benchmark_test.go

```diff
@@ -613,14 +627,30 @@ func BenchmarkJetStreamPublish(b *testing.B) {
 								b.Logf("Creating stream with R=%d and %d input subjects", bc.replicas, bc.numSubjects)
 							}
 							streamConfig := &nats.StreamConfig{
-								Name:     "S",
+								Name:     streamName,
 								Subjects: subjects,
 								Replicas: bc.replicas,
 							}
 							if _, err := js.AddStream(streamConfig); err != nil {
 								b.Fatalf("Error creating stream: %v", err)
 							}
 
+							// cluster_size > 1, connect to stream leader
+							if cl != nil {
+								connectURL = cl.streamLeader("$G", streamName).ClientURL()
```

Comment:
The one above uses jsConnectURL which does error checking. Any reason for not doing the same here?

Reply:
There is additional `JetStream.opts` that are used in this benchmark unlike the one above. Is the error checking here not sufficient?